mucommander-core/src/main/java/com/mucommander/ui/main/WindowManager.java

```diff
@@ -129,11 +119,39 @@ private WindowManager() {
             LOGGER.debug("Could load look'n feel from preferences");
         
         MuConfigurations.addPreferencesListener(this);
+
+        if (!additionalLafsLoaded) {    // defer loading other LAFs
```

Comment:
in which scenario would we get here with additionalLafsLoaded==true?

Reply:
line 116: setLookAndFeel(lnfName), a couple lines above. This is case when chosen LnF is from "additional ones" - then loading is being forced (not deferred).